dev-haskell/fay/fay-0.24.2.0.ebuild

```diff
@@ -16,9 +16,11 @@ LICENSE="BSD"
 SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86"
 IUSE="test"
-# See [issue upstream][1] for resolving circular dependency
+# See [issue upstream][1] for fixing tests. Currently, they have resolved the
+# circular dependency (yay!) but most tests fail (booo) due to `ghc-pkg: cannot
+# find package fay-base`
```

Comment:
I don't understand the comment. Do tests work or do they not? You comment out the `RESTRICT=test` below.

Reply:
I'll need to fix this commit (i believe tests should be restricted) and improve the message.

The original issue was a circular dependency. After filing a bug, upstream fixed the dependency, but now most tests fail due to other upstream problems.

Again, I'll clean this up.

Reply:
Check out the updated comment, let me know if it is more clear. If so, I will mark this conversation 'resolved'